File: Python/program26.py
def display(i_frequency: int):
    for i_cnt in range(1, i_frequency + 1):
        print("Jay Ganesh...")

def main():
    try:
        i_count = int(input("Enter the frequency : "))
        display(i_count)
    except ValueError:
        print("Please enter a valid integer.")

if __name__ == "__main__":
    main()
    

File: Java/Program26.java
class program23
{
    public static void main (String A[])
    {
        int iNo = 0;
        program23 idobj = new program23();
        idobj.display(iNo);
        
    }

    public void display (int iNo)
    {
        if (iNo <= 0)
        {
            System.out.println("Please enter positive number");
        }
        else
        {
        for (int i = 1; i<iNo ;i++)
            {
                System.out.println("Jay Ganesh....");
            }
        }

    }
}